app/models/publication.rb

```diff
@@ -488,6 +494,12 @@ def doi_url_path
       field(:edition)
       field(:page_range)
       field(:doi) { label 'DOI' }
+      field(:doi_verified, :enum) do
+        label 'DOI verified?'
+        enum do
+          [['True', true], ['False', false]]
+        end
+      end
```

Comment:
I'm assuming that leaving the field blank (null) here is an option too and that's why you used an enum instead of a checkbox, correct?

Reply:
Correct.  Also, if an admin were to make any edits to a publication with a doi_verified as `nil`, the boolean would change from `nil` to `false` on update, which could throw things off.